packages/warriorjs-core/src/Unit.js

```diff
@@ -10,11 +10,13 @@ class Unit {
    * @param {string} character The character of the unit.
    * @param {number} maxHealth The max health in HP.
    * @param {boolean} captive Whether the unit is a captive or not.
+   * @param {number} reward The number of points to reward when killed.
    */
-  constructor(name, character, maxHealth, captive = false) {
+  constructor(name, character, maxHealth, captive = false, reward = null) {
```

Comment:
@RascalTwo If you put the `reward` parameter before `captive` it could make the tests easier because you won't have to worry about passing a value for `captive`, see comment in the test.